src/pages/all-on-one/all-on-one.ts

```diff
@@ -20,6 +20,9 @@ import { CustomHammerConfigProvider } from '../../providers/custom-hammer-config
 export class AllOnOnePage {
   isDButtonPressed = false;
   isSButtonPressed = false;
+  isManoeuverEnabled = false
+
+  @ViewChild('maneuversButton') maneuversButton
```

Comment:
'maneuvers' is spelt in American English here but spelt in International English ('manoeuvres') elsewhere in the app.